packages/python-sdk/tests/sync/sandbox_sync/files/test_write.py

```diff
@@ -119,3 +120,25 @@ def test_write_to_non_existing_directory(sandbox):
 
     read_content = sandbox.files.read(filename)
     assert read_content == content
+
+def test_write_with_secured_envd(template):
+    filename = "non_existing_dir/test_write.txt"
```

Comment:
can you also do something like (random), so if you test envd locally it doesn't interfere 
```
    dir_name = f"test_directory_{uuid.uuid4()}"
 ```